src/styles/reset.css

```diff
@@ -0,0 +1,3 @@
+.reset-margin {
+  margin: 0;
+}
```

Comment:
in normalize file => 

* {
  box-s: boeder-box;
}

html {
 f-f: kldsmkf
}